core/src/main/java/edu/softserve/zoo/util/ListUtils.java

```diff
@@ -0,0 +1,27 @@
+package edu.softserve.zoo.util;
+
+import edu.softserve.zoo.exceptions.ApplicationException;
+
+import java.util.List;
+
+/**
+ * <p>This utility class helps to work with lists</p>
+ *
+ * @author Taras Zubrei
+ */
+public class ListUtils {
```

Comment:
This class shouldn't be here. If you have findOne method in repo, this method is redundant.